lib/gs-plugin-job.c

```diff
@@ -67,6 +68,7 @@ enum {
 	PROP_MAX_RESULTS,
 	PROP_PRICE,
 	PROP_TIMEOUT,
+	PROP_DEST,
```

Comment:
I've fixed the name locally.

I want the flexibility to change the destination on a per-job basis, I think, since the path can change as USB disks are added and removed and, in the future, I'll want to add the ability for the user to select the destination. Right now, it just picks the most-recently-mounted removable disk which is kind of crude.

Given those needs, would it still make sense to do this in the plugin loader?